src/meshes/triangle_mesh.template.cc

```diff
@@ -34560,10 +34566,6 @@ namespace oomph
         this->copy_connection_information(polygon_pt->polyline_pt(p),
                                           tmp_curve_section_pt);
 
-        // Now update the polyline according to the new vertices but
```

Comment:
Is this now redundant? Looks a bit dangerous to remove, unless the potential memory leak is addressed elsewhere.

Reply:
not yet! I've just pushed the change to the mpirun flag. May as well wait until the github ci tests pass for that too.

Reply:
OK. All Approved.